recipe/meta.yaml

```diff
@@ -10,19 +10,24 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/HowardHinnant/date/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - use-conda-tzdb-on-all-platforms.patch 
 
 build:
-  number: 3
+  number: 4
   skip: true  # [win and vc<14]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - {{ stdlib('c') }}
```

Comment:
Is there a reason for removing this?

Reply:
It does not work when running conda-build from osx-arm platform. Happy to re-add.